.gitlab-ci-github-status-updates.yml

```diff
@@ -13,6 +13,8 @@ stages:
 
 github:start:
   stage: .pre
+  except:
+    - /^(master|staging|[0-9]+\.[0-9]+\.x)$/
```

Comment:
I think we should keep it running here. This is occasionally useful information. For example, after the nightly has run, the status will correctly show up on that commit.

Reply:
Okey, that is a valid point. I will then override these only for mender-qa pipeline (expect a new PR later)